src/core/server/saved_objects/migrations/core/build_active_mappings.test.ts

```diff
@@ -91,6 +92,12 @@ describe('buildActiveMappings', () => {
     expect(hashes.aaa).toEqual(hashes.bbb);
     expect(hashes.aaa).not.toEqual(hashes.ccc);
   });
+
+  test('workspaces field is added when workspace feature flag is enabled', () => {
```

Comment:
Do we need to add a test for when the workspace feature flag is not enabled, and the workspace field does not exist?

Reply:
In previous test cases I think they've already cover this case but will take a look to confirm.

Reply:
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/core/server/saved_objects/migrations/core/__snapshots__/build_active_mappings.test.ts.snap#L4

This snapshot should cover the case when feature flag is not enabled.